### DISCUSSION TITLE
Have the buttons been removed?

### DISCUSSION THREAD

**Discussion by RoboEverest** [maximmax42/Discord-CustomRP#246] [General]
I noticed that after the last update the buttons were removed, is that right or is something wrong here on my discord?

**Comment by maximmax42** [comment]
Discord updated mini-profile's design and now people can't see their own buttons. Buttons still work, everyone but you will see them. Nothing I can do.

**Reply by RoboEverest** [reply]
I noticed that the profile has gotten smaller.
Even so, thanks for the reply.
But in the future, could these buttons perhaps reappear?

**Reply by maximmax42** [reply]
Probably, if Discord will consider this a bug and will actually fix it.

**Reply by RoboEverest** [reply]
Great.
Thanks, friend. Have a good day.
